catalog/MDCCatalog/MDCCatalogComponentsController.swift

```diff
@@ -103,7 +103,11 @@ class MDCCatalogComponentsController: UICollectionViewController, MDCInkTouchCon
     let titleLabel = UILabel()
     titleLabel.text = self.title!.uppercased()
     titleLabel.textColor = UIColor(white: 1, alpha: 1)
-    titleLabel.font = UIFont(name: "RobotoMono-Regular", size: 14)
+    if #available(iOS 9.0, *) {
+        titleLabel.font = UIFont.monospacedDigitSystemFont(ofSize: 14, weight: UIFontWeightRegular)
+    } else {
+        titleLabel.font = UIFont().withSize(14)
```

Comment:
This is crashing when I run on iOS 8.3 simulator. It seems that UIFont() doesn't actually allocate a font (?)

```
2017-09-14 11:14:49.326 MDCCatalog[86183:953757] -[UIFont ctFontRef]: unrecognized selector sent to instance 0x7a694d90
2017-09-14 11:14:49.329 MDCCatalog[86183:953757] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UIFont ctFontRef]: unrecognized selector sent to instance 0x7a694d90'
*** First throw call stack:
(
	0   CoreFoundation                      0x0104a746 __exceptionPreprocess + 182
	1   libobjc.A.dylib                     0x00cd3a97 objc_exception_throw + 44
	2   CoreFoundation                      0x01052705 -[NSObject(NSObject) doesNotRecognizeSelector:] + 277
	3   CoreFoundation                      0x00f99287 ___forwarding___ + 1047
	4   CoreFoundation                      0x00f98e4e _CF_forwarding_prep_0 + 14
	5   CoreText                            0x056ba045 CTFontGetClientObject + 78
	6   GraphicsServices                    0x04fae75e GSFontEnsureExtraData + 29
	7   GraphicsServices                    0x04fb2463 GSFontGetExtraData + 41
	8   UIFoundation                        0x045f6c5d -[UIFont isSystemFont] + 22
	9   UIFoundation                        0x045f6ad7 -[UIFont fontWithSize:] + 36
	10  MDCCatalog                          0x00082760 _T010MDCCatalog0A20ComponentsControllerC11viewDidLoadyyF + 1744
```

Reply:
Yeah, I just saw this in a playground fixing it now.